src/ui.form-calendar/UCalendar.vue

```diff
@@ -336,13 +339,18 @@ watch(
 watch(
   selectedDate,
   () => {
-    if (selectedDate.value && isTimepickerEnabled.value && isInputRefs.value && props.timepicker) {
-      hoursRef.value!.value = String(selectedDate.value.getHours()).padStart(2, "0");
-      minutesRef.value!.value = String(selectedDate.value.getMinutes()).padStart(2, "0");
-      secondsRef.value!.value = String(selectedDate.value.getSeconds()).padStart(2, "0");
-
-      emit("userDateChange", userFormattedDate.value);
-    }
+    nextTick(() => {
+      if (
+        selectedDate.value &&
+        isTimepickerEnabled.value &&
+        isInputRefs.value &&
+        props.timepicker
```

Comment:
move to 1 or several variables with clear name

Reply:
I've just removed props.timepicker. It's okay in one line I guess